xdsl/dialects/arm_neon.py

```diff
@@ -7,6 +7,7 @@
 from xdsl.dialects.arm.ops import ARMInstruction, ARMOperation
 from xdsl.dialects.arm.register import ARMRegisterType, IntRegisterType
 from xdsl.dialects.builtin import IntegerAttr, StringAttr, i8
+from xdsl.dialects.stencil import IndexAttr
```

Comment:
This import looks suspicious

Reply:
yeah thanks for catching that, deleting it now